README.md

```diff
@@ -37,6 +40,21 @@ malware and what it does.
 
 As of the time of writing this new control IP has been taken offline by its hosting provider.
 
+### Curseforge
+
+Curseforge is fully aware of this ongoing problem and peforming their own investigation. 
+You can see their latest updates in their [Discord](https://discord.com/invite/curseforge) server.
+
+Their most recent announcement was at `1686153780` or Wednesday June 07 2023 16:03:00 GMT+0000, and is as follows follows:
+> 📢 UPDATE: Repeated Questions
+> 
+> Hey all, as we are still seeing some misconceptions regarding the current situations, please note the following:
+> 
+> ONLY Minecraft is affected. This malware specifically targets Minecraft JAR files and nothing else. Any concerned players can rest assured their Addons, CC and non Minecraft mods are safe.
```

Comment:
According to https://github.com/fractureiser-investigation/fractureiser/pull/26#discussion_r1222184746 this statement is wrong!